pytorch_lightning/loops/training_epoch_loop.py

```diff
@@ -347,7 +353,7 @@ def _prepare_outputs(
             processed_outputs = processed_outputs[0]
         return processed_outputs
 
-    def update_lr_schedulers(self, interval: str) -> None:
+    def update_lr_schedulers(self, interval: str, update_plateau_schedulers: bool = False) -> None:
```

Comment:
The new parameter should be either `Optional[bool]` to update both or a required argument. Leaning towards the second option